main.py

```diff
@@ -153,11 +153,85 @@ def load_participants(self):
                 page_token = response.get("next_page_token")
                 params["next_page_token"] = page_token
 
+    def load_groups(self):
+        table_name = "Zoom_Groups"
+        response = self.client.group.list().json()
+        if response:
+            self.drop_table(table_name)
+        results = response.get("groups")
+        if results:
+            groups = pd.DataFrame(results)
+            self.sql.insert_into(table_name, groups)
+            logging.info(f"Inserted {len(groups)} records into {table_name}")
+
+    def _get_group_ids(self, groupname=None):
+        """Get list of group ids to iterate API calls for group members"""
+        groups = pd.read_sql_table(
+            table_name="Zoom_Groups", con=self.sql.engine, schema=self.sql.schema
+        )
+        if groupname:
+            groups = groups[groups["name"] == groupname]
+        group_ids = groups["id"].values.flatten().tolist()
+        return group_ids
+
+    def load_group_members(self):
+        """Load Zoom group member data"""
+        table_name = "Zoom_GroupMembers"
+        self.drop_table(table_name)
+        for group_id in self._get_group_ids():
+            page = 1
+            params = {"groupid": group_id, "page_size": 300, "page_number": page}
+            response = self.client.group.list_members(**params).json()
+            page_count = response["page_count"]
+            while page <= page_count:
+                response = self.client.group.list_members(**params).json()
+                results = response.get("members")
+                if results:
+                    members = pd.DataFrame(results)
+                    members["groupId"] = group_id
+                    self.sql.insert_into(table_name, members)
+                    logging.info(f"Inserted {len(members)} records into {table_name}")
+                    page += 1
+                    params["page"] = page
+
+    def _get_students(self):
+        """Query database for new students without Zoom accounts"""
+        return pd.read_sql_table(
+            "vw_Zoom_NewStudentAccounts", con=self.sql.engine, schema=self.sql.schema
```

Comment:
Should this view definition be added to the repo?

Reply:
It will be in sz_custom